### DISCUSSION TITLE
Long wave output missing in fluxes.csv

### DISCUSSION THREAD

**Discussion by mone27** [eap-fes-unigoe/canopy-model#8] [General]
@femoyano in the 2018 fluxes file there `lw_out` variable is always `NA` and we would need to for the model calibration. In the first data that we received there were `lw_out` values. Is there a reason behind this NAs or should I fixe the fluxes.csv file?

**Comment by femoyano** [comment]
@mone27 I thought it was all NA because they were missing values, but maybe I did something that deleted the data. Not on purpose. Please fix it if you can.
